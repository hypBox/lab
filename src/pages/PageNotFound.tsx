import React from "react"

const pageNotFound = () => {
  return <h1>404 Page Not Found</h1>
}

export default pageNotFound
