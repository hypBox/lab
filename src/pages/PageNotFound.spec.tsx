import React from "react"
import renderer from "react-test-renderer"

import PageNotFound from "./PageNotFound"

describe("Page Not Found", () => {
  it("renders as expected", () => {
    const tree = renderer.create(<PageNotFound />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
