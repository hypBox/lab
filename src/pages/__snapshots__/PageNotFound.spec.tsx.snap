// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard renders as expected 1`] = `
<h1>
  404 Page Not Found
</h1>
`;
